prettify.js

```diff
@@ -1,33 +1,80 @@
 require("./prettify.less");
 
+require("./prism-config");
 var Prism = require("prismjs");
 require("prismjs/plugins/line-numbers/prism-line-numbers.js");
 require("prismjs/plugins/previewers/prism-previewers.js");
 require("prismjs/plugins/command-line/prism-command-line.js");
+require("prismjs/plugins/line-highlight/prism-line-highlight.js");
+require("prismjs/plugins/toolbar/prism-toolbar.js");
+require("prismjs/plugins/copy-to-clipboard/prism-copy-to-clipboard.js");
+require("./prism-collapse.js");
 
 require("prismjs/themes/prism-coy.css");
 require("prismjs/plugins/line-numbers/prism-line-numbers.css");
 require("prismjs/plugins/previewers/prism-previewers.css");
 require("prismjs/plugins/command-line/prism-command-line.css");
+require("prismjs/plugins/line-highlight/prism-line-highlight.css");
+require("prismjs/plugins/toolbar/prism-toolbar.css");
+require("./prism-collapse.less");
 
 /**
- * @parent bit-docs-prettify/static
- * @module {function} bit-docs-prettify/prettify.js
- *
- * Main front end JavaScript file for static portion of this plugin.
- *
- * @signature `prettyPrint()`
- *
- * Finds all `<code>` elements on the page and adds the `prettyprint` class
- * before executing the required pretty print engine.
- *
- * Also requires [bit-docs-prettify/prettify.less].
+ * Get node for provided line number
+ * Copied from prism-line-numbers.js and modified to support nested spans
+ * @param {Element} element pre element
+ * @param {Number} number line number
+ * @return {Element|undefined}
  */
-module.exports = function(){
+Prism.plugins.lineNumbers.getLine = function (element, number) {
```

Comment:
can you add some docs explaining this?

Reply:
there was a bit, but I added more to the docblock above. also, this has been moved to `bit-docs-html-highlight-line`